.pre-commit-config.yaml

```diff
@@ -3,7 +3,7 @@ repos:
     rev: 21.9b0
     hooks:
       - id: black
-        exclude: ^poetry/core/_vendor
+        exclude: ^src/poetry/core/_vendor
```

Comment:
unrelated to this pr, but `^src/poetry/core/_vendor` could be moved to the [top-level `exclude`](https://pre-commit.com/#pre-commit-configyaml---top-level)

Reply:
ack, we can do that in another PR. I think @neersighted is doing a general cleanup once the config for `poetry` stabilises.

Reply:
I'm personally fine leaving it for now until we start bringing poetry's `.pre-commit-config.yaml` over.